frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -552,6 +571,12 @@ export function Experiment(): JSX.Element {
                                 <Col span={19} style={{ color: '#f96132' }}>
                                     Your results are <b>not statistically significant</b>. We don't recommend ending
                                     this experiment yet.
+                                    <Tooltip
+                                        placement="right"
+                                        title="This can be because the number of people exposed to the experiment is less than 100, or the results are not statistically significant."
```

Comment:
Hmmm, now that you say it, your suggestion is a lot better, but it's a bit more weirder than that, since:

The banner says: "results are not statistically significant"

But, the banner can say the above for 3 different cases: 
1. < 100 peeps
2. probability < 90%
3. Stat sig calculations say results are not significant

(ref: https://posthog.com/docs/user-guides/experimentation#important-note-about-significance ). Only when all three are satisfied does the banner turn green.

@clarkus for perhaps more input here? How can we best explain this?

Since, if they check the tooltip saying <100 peeps exposed, and they can see they have more people exposed, boom confusion xP

Reply:
What if we give the user a likely explanation for why their results are not significant instead of a general tooltip? Since we have the data on hand anyway

Reply:
Anyway I think the rest of the changes in this PR look good, feel free to merge and can always iterate on that ^

Reply:
Yeah, that makes more sense. Either less than 100, or 'expected loss > 1%' or 'p value > 0.05'. Nice, much more detailed.

I'll update the backend to send more info about significance & exposure, and then use that to determine things here.